storage-s3-dynamodb/src/main/java/io/delta/storage/BaseExternalLogStore.java

```diff
@@ -396,7 +401,13 @@ private void fixDeltaLog(FileSystem fs, ExternalCommitEntry entry) throws IOExce
                     fixDeltaLogPutCompleteDbEntry(entry);
                     LOG.info("fixed file {}", entry.fileName);
                     return;
-                } catch(Throwable e) {
+                } catch (java.nio.file.FileAlreadyExistsException e) {
```

Comment:
The false FileAlreadyExistsException comes from `io.delta.storage.BaseExternalLogStore#copyFile` which is called by `fixDeltaLogCopyTempFile` and `writeCopyTempFile`. Could you fix the cases for `writeCopyTempFile` as well?

Reply:
We already catch and ignore any exception when using `writeCopyTempFile`. I can add it there but all it will really do is slightly alter the error message that we log.

Reply:
I see.